src/handlers/v1/markets/[chainId]/[marketId]/collateral-swap/index.ts

```diff
@@ -60,33 +60,33 @@ export const v1GetCollateralSwapQuotationRoute: Route<GetCollateralSwapQuotation
     const { utilization, healthRate, liquidationThreshold, borrowUSD, collateralUSD, netAPR } = marketInfo;
     const currentPosition = { utilization, healthRate, liquidationThreshold, borrowUSD, collateralUSD, netAPR };
 
-    let targetTokenAmount = '0';
+    let destAmount = '0';
     const logics: GetCollateralSwapQuotationResponseBody['logics'] = [];
     let fees: GetCollateralSwapQuotationResponseBody['fees'] = [];
     let approvals: GetCollateralSwapQuotationResponseBody['approvals'] = [];
     let targetPosition = currentPosition;
-    if (event.body.withdrawalToken && event.body.targetToken && event.body.amount && Number(event.body.amount) > 0) {
-      const { amount, slippage } = event.body;
+    if (event.body.srcToken && event.body.destToken && event.body.srcAmount && Number(event.body.srcAmount) > 0) {
+      const { srcAmount, slippage } = event.body;
       const { supplyAPR, supplyUSD, borrowAPR, borrowCapacityUSD, liquidationLimit, collaterals } = marketInfo;
 
       // Verify token input
```

Comment:
```suggestion
      // Verify token input
```

Reply:
Anything need to change here?

Reply:
I'd like to remove the comment since our current implementation doesn't have a token input concept.